### DISCUSSION TITLE
Passthrough templater syntax

### DISCUSSION THREAD

**Discussion by twiddli** [SilentVoid13/Templater#1140] [Help] [answered]
Hi, I'm trying to create a template that lets some templater syntax stay untouched when triggered so the resulting note keeps the syntax.

Anyway to accomplish this?

This doesn't work:
```
<% "<% 'test' %>" %>
```

**Comment by twiddli** [comment, marked as answer]
Thought up a solution right after hitting submit. Classic.

I ended up creating a user script named `utils.js` with the following function:
```js
class Utils {
  function t(content, type = "+") {
      return `<%${type} ${content} %>`;
    }
}
const utils = new Utils();
exports.default = () => utils;
```

It's important that this resides in a user script or it won't work.

Now I can use it like this inside a template note

```js
<% tp.user.utils().t("'test'") %>
```

Which results in a note with
```js
<%+ 'test' %>
```

The type of command can be specified with a second argument: `<% tp.user.utils().t("'test'", "*") %>` for `<%* 'test' %>`

**Comment by Zachatoo** [comment]
You can escape the `<` characters with `\`

```
<% "\<% 'test' %\>" %>
```
